FEXCore/Source/Interface/Core/JIT/Arm64/JIT.cpp

```diff
@@ -84,9 +84,14 @@ void Arm64JITCore::Op_Unhandled(IR::IROp_Header const *IROp, IR::NodeID Node) {
   } else {
     switch(Info.ABI) {
       case FABI_F80_I16_F32:{
-        SpillStaticRegs(TMP1);
+        if (Info.SupportsPreserveAllABI) {
```

Comment:
probably want to factor out a helper for this

Reply:
Added helper